src/commonMain/kotlin/exchange.dydx.abacus/utils/String+Utils.kt

```diff
@@ -24,6 +24,29 @@ fun String.toNobleAddress(): String? {
     }
 }
 
+fun String.toOsmosisAddress(): String? {
```

Comment:
ask skip about how fallback addresses work - do we need to add these wallets to our autosweeping logic?

Reply:
confirmed it's at parity. they do not automatically sweep back into noble but not worried about that rn